### PR TITLE
Closes #88 2.11 fill in profile before application

### LINKED ISSUE #88: US_2.11: Fill In Profile Before Application
#**As**
organizer

**I want**
pupils to fill in their profile before applying to anything

**In order to**
make sure that all necessary data is provided

**Dependencies:** #92

**Estimate:** 5

**Acceptance Criteria:**
- [ ] display an error message when user tries to apply for something and mandatory profile fileds are not filled
- [ ] text of error message: "Du musst erst dein Profil ausfüllen, um dich für Workshops bewerben zu können. [Zum Profil](https://en.wikipedia.org/wiki/Profil)"
- [ ] link in error takes user to profile
- [ ] display error on same page
- [ ] works for all apply-button

### PR DESCRIPTION
**Pull request by cmfcmf**
@hefowe

### CODE REVIEW

File: config/locales/de.application_letters.yml [hunk 1]
```diff
@@ -26,7 +27,7 @@ de:
     attributes:
       application_letter:
           grade: "Klassenstufe"
-          experience: "Wie hast du von diesem Event erfahren?"
+          experience: "Wie hast du von uns Erfahren?"
```
Comment (MasterCarl) [inline config/locales/de.application_letters.yml]: I'd use a lowercase e

File: spec/features/application_letters_spec.rb [hunk 2]
```diff
@@ -95,6 +92,23 @@
     expect(page).to_not have_text login_error_message
   end
 
+  it "shows an error if you don't have a profile" do
```
Comment (MasterCarl) [inline spec/features/application_letters_spec.rb]: I'd also mention that a redirect should happen

**Review by MasterCarl — CHANGES_REQUESTED** [review]

**Review by MasterCarl — APPROVED** [review]

**Comment by MasterCarl** [comment]
Maybe someone from team-hendrik could add their review as well

**Comment by cmfcmf** [comment]
> Maybe someone from team-hendrik could add their review as well

Whoops, forgot about this. Thank you @MasterCarl, sorry @hefowe. @nilsthamm should take a look.

**Review by nilsthamm — APPROVED** [review]

**Pull request merged**